src/scripts/cfo.zig

```diff
@@ -240,6 +225,166 @@ fn run_fuzzers(
+    // different branches (to fuzz PRs and releases).
+    shell.project_root.deleteTree("working") catch {};
+
+    { // Main branch fuzzing.
+        const commit = try run_fuzzers_commit_info(shell);
+        log.info("fuzzing commit={s} timestamp={d}", .{ commit.sha, commit.timestamp });
+
+        try shell.exec("git clone https://github.com/tigerbeetle/tigerbeetle working/main", .{});
+        try shell.pushd("./working/main");
+        defer shell.popd();
+
+        try shell.exec("git switch --detach {commit}", .{ .commit = @as([]const u8, &commit.sha) });
+        shell.zig("build -Drelease build_fuzz", .{}) catch |err| {
+            log.warn("failed to build: {}", .{err});
+        };
+
+        for (std.enums.values(Fuzzer)) |fuzzer| {
+            try working_directory.append("./working/main");
+            try seed_record.append(.{
+                .commit_timestamp = commit.timestamp,
+                .commit_sha = commit.sha,
+                .fuzzer = fuzzer,
+                .branch = "https://github.com/tigerbeetle/tigerbeetle",
+            });
+        }
+    }
+    const task_main_count: u32 = @intCast(seed_record.items.len);
+
+    if (gh_token != null) {
+        // Any PR labeled like 'fuzz lsm_tree'
+        const GhPullRequest = struct {
+            const Label = struct {
+                id: []const u8,
+                name: []const u8,
+                description: []const u8,
+                color: []const u8,
+            };
+            number: u32,
+            labels: []Label,
+        };
+
+        const pr_list_text = try shell.exec_stdout(
+            "gh pr list --state open --json number,labels",
+            .{},
+        );
+        const pr_list = try std.json.parseFromSliceLeaky(
+            []GhPullRequest,
+            shell.arena.allocator(),
+            pr_list_text,
+            .{},
+        );
+
+        for (pr_list) |pr| {
+            for (pr.labels) |label| {
+                if (stdx.cut(label.name, "fuzz ") != null) break;
+            } else continue;
+
+            const pr_directory = try shell.print("./working/{d}", .{pr.number});
+
+            try shell.exec(
+                "git clone https://github.com/tigerbeetle/tigerbeetle {directory}",
+                .{ .directory = pr_directory },
+            );
+
+            try shell.pushd(pr_directory);
+            defer shell.popd();
+
+            try shell.exec(
+                "git fetch origin refs/pull/{pr_number}/head",
+                .{ .pr_number = pr.number },
+            );
+            try shell.exec("git switch --detach FETCH_HEAD", .{});
+
+            const commit = try run_fuzzers_commit_info(shell);
+            log.info("fuzzing commit={s} timestamp={d}", .{ commit.sha, commit.timestamp });
+
+            shell.zig("build -Drelease build_fuzz", .{}) catch |err| {
+                log.warn("failed to build: {}", .{err});
+            };
```

Comment:
To check my understanding -- is the `zig build build_fuzz` here (and above) to prevent all of the child fuzz processes from trying to compile the same code concurrently?

Reply:
Not exactly: it wouldn’t be wrong to just remove these.

But it is helpful to:

* get more precise times for seed (otherwise, the first seed would seem longer than it is)
* log “this failed to compile” for cfo itself. CFO doesn’t really try to understand *why* things fail, but compilation failures feel worthwhile to log, just in cas something abut out build process itself changes.